new/LOC7374Armenia.xml

```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LOC7374Armenia" xml:lang="en" type="place">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Armenia</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Country in the Armenian highlands and Caucasus region.</p>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+                <language ident="hy">Armenian</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-05-31">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+                <body>
+                    <listPlace>
+                        <place sameAs="wd:Q208404" type="country">
```

Comment:
Kingdom of Armenia = a kingdom in the Ancient Near East which existed **from 331 BC to 428 AD**
I do not think it fits

Reply:
I would take the broad and neutral ID  [Q399)](https://www.wikidata.org/wiki/Q399) 
(besides, in our medieval context we usually speak about Armenian Kingdom of Cilicia, https://www.wikidata.org/wiki/Q335088 which is actually in present-day Turkey, but that was the "Armenia" where Eustacius went to)

Reply:
It may cause confusion. I think, it is better to update the abstract as well, so it is more clear, that we deal with modern day Armenia here.

Reply:
Correct, I suggested Q399 which covers everything.

Reply:
It corresponds to https://en.wikipedia.org/wiki/Armenia covering the whole region and history

Reply:
I understand. We should keep this strategy for all similar cases. This is why I encoded Q115 to LOC3010Ethiop. Do you think, this is correct?

Reply:
I think, in many cases, there will be an ambiguity, just because it is often not specified to which entity a country refers. For many cases, we must determine, which relation is dominant. The text in BMLacq680 might be a rather modern text in Amharic that was authored before the national state of Armenia was founded. I think, the theologician pointed more on religious or national cultural aspects in general. Nevertheless I would keep the relation to Q399, because we do not have an ID for every aspect and this seems to me the closest. For medieval Armenia in Cilicia, we might need another ID. In every case, this is likely to cause a lot of confusion for encoders and users in the future.